calico/openstack/test/test_plugin_etcd.py

```diff
@@ -764,3 +946,45 @@ def _neutron_rule_from_dict(overrides):
     }
     rule.update(overrides)
     return rule
+
+class TestIfAgentState(object):
+    """
+    Class to help decide if object is agent_state
+
+    :param: kwargs for agent attributes.
+    """
+    def __init__(self, agent_type=None, binary=None, host=None, topic=None, start_flag=None):
+        self.agent_type = agent_type
+        self.binary = binary
+        self.host = host
+        self.topic = topic
+        self.start_flag = start_flag
+    def __eq__(self, other):
+        if type(other) != dict:
+            return False
+        if self.agent_type != None and other["agent_type"] != self.agent_type:
+            return False
+        if self.binary != None and other["binary"] != self.binary:
+            return False
+        if self.host != None and other["host"] != self.host:
+            return False
+        if self.topic != None and other["topic"] != self.topic:
+            return False
+        if self.start_flag and not other.get("start_flag"):
+            return False
+        if self.start_flag == False and other.get("start_flag"):
+            return False
+        return  True
```

Comment:
Nitpick: extra space, and there should be a newline separating these two methods.